### PR TITLE
fix: #8437 Fixes QTree lazy-loading issues

### PR DESCRIPTION
**Pull request by suyundukov**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**
Closes https://github.com/quasarframework/quasar/issues/8437

### CODE REVIEW

**Comment by rstoenescu** [comment]
Hi,

Thanks for contributing!
It would be better if the initial problem is solved, which is setting the "localLazy" prop on "m" instead of "lazy: localLazy".

**Comment by nurlansu** [comment]
> It would be better if the initial problem is solved, which is setting the "localLazy" prop on "m" instead of "lazy: localLazy".

Yeah, you are right. That's much better.

File: ui/src/components/tree/QTree.js [hunk 1]
```diff
@@ -329,8 +329,7 @@ export default defineComponent({
             lazy.value[ key ] = 'loaded'
             node[ props.childrenKey ] = Array.isArray(children) === true ? children : []
             nextTick(() => {
-              const localMeta = m.value[ key ]
-              if (localMeta && localMeta.isParent === true) {
+              if (m && m.isParent === true) {
```
Comment (nurlansu) [inline ui/src/components/tree/QTree.js:332]: For some reason `isParent` doesn't get recalculated after setting node's children.

Can someone help me with that, sadly I don't really understand what's the problem here.

Comment (nurlansu) [reply ui/src/components/tree/QTree.js]: My bad, fixed now.

**Comment by rstoenescu** [comment]
Nicely done!
This will go into beta.4.

**Pull request merged**